gulpfile.js/tasks/rev/rev-font-workaround.js

```diff
@@ -54,3 +58,11 @@ gulp.task('rev-font-workaround', ['rev-assets'], function() {
 
   return merge.apply(this, streams);
 });
+
+function exists(path) {
+  try {
+    return fs.openSync(filePath, 'r')
+  } catch(e) {
+    return false
+  }
+}
```

Comment:
I don't think this is quite the right approach. The only thing this task should be doing is copying the hash off of the iconFont.svg file, and adding it to the corresponding woff, eot, and ttf icon font files. I'd leave this file as is, and instead, be more specifc in the "not these" list in `rev-assets.js`, allowing non-iconFont files to be revved.